### PR TITLE
PLAT-7777 Manual Synch Courses

### PR DESCRIPTION
**Pull request by jadkhateradmin**
https://administrate.atlassian.net/browse/PLAT-7777

### CODE REVIEW

File: assets/js/admin-debug.js [hunk 1]
```diff
@@ -353,6 +353,24 @@
       }
     );
 
+    $(defaults.importCoursesBtn).on(
+      'click',
+      function(){
+
+        var button = $(this);
+
+        if (button.hasClass(baseDefaults.loaddingClass)) {
```
Comment (alihabib-administrate) [inline assets/js/admin-debug.js]: loadingClass has a spelling error

File: assets/js/admin-debug.js [hunk 2]
```diff
@@ -401,7 +419,57 @@
...
+        }
+      });
+    },
+
+    importCourses: function(button) {
+      var self = this;
+      var defaults = $.ADMSettings.defaults;
+      var baseDefaults  = $.ADMBase.defaults;
+
+      if (button.hasClass(baseDefaults.loaddingClass)) {
```
Comment (alihabib-administrate) [inline assets/js/admin-debug.js]: here too

**Review by alihabib-administrate — CHANGES_REQUESTED** [review]
loadingClass is spelled loaddingClass everywhere

File: assets/js/admin-debug.js (hunk 2 above)
Comment (jadkhateradmin) [reply assets/js/admin-debug.js]: Done

File: assets/js/admin-debug.js (hunk 1 above)
Comment (jadkhateradmin) [reply assets/js/admin-debug.js]: Done

**Review by alihabib-administrate — APPROVED** [review]

**Review by alihabib-administrate — APPROVED** [review]
LGTM

**Pull request merged**